doonut-app-external-api/src/main/kotlin/com/doonutmate/oauth/kakao/dto/KakaoInfoResponse.kt

```diff
@@ -1,32 +1,16 @@
 package com.doonutmate.oauth.kakao.dto
 
 import io.swagger.v3.oas.annotations.media.Schema
-@Schema(title = "더미용 DTO", description = "추후 들어올 정보에 따라 구조 달라짐, 테스트용 더미 반환 DTO")
+
+@Schema(title = "카카오 토큰을 이용해, 카카오 oauth 서버에서 반환받는 정보들을 담은 dto")
 data class KakaoInfoResponse(
-    val id: Long?,
+    // TODO: 이거 long으로 바꿔야함
+    val id: String?,
```

Comment:
id가 null이면 그냥 에러가 발생하도록 하는게 좋을 것 같아요.

```suggestion
    // TODO: 이거 long으로 바꿔야함
    val id: String,
```

Reply:
궁금한게 이건 왜 String이 아닌 long으로 바꿔야하나요?

Reply:
이 부분은 카카오에서 받아온 **ouathId**가 **Long**으로 넘어옵니다. 일단은 **member record**의 **oauthId**가 **string**이라서 **KakaoInfoResponse DTO**도 **string**으로 만들두었습니다. **string**여도 문제는 안생기긴 하는데 제가 모르는 부분에서 문제가 생길 거 같아서 넘어온 타입을 맞춰주는 게 맞다고 생각했습니다!!

Reply:
아하 그렇다면 Long 가도 상관없을 것 같네요.  
전 카카오의 정책 변경에 따라 oauthId가 Long에서 String으로 바뀌게 되면 저희도 바꾸어야 하기에 안전하게 String으로 저장하는게 낫지 않을까? 생각했습니다ㅎ